server/src/main/webapp/WEB-INF/rails/webpack/views/components/buttons/index.tsx

```diff
@@ -88,18 +92,27 @@ abstract class Button extends RestyleViewComponent<Styles, Attrs> {
 
   abstract type(): string;
 
+  oncreate(vnode: m.Vnode<Attrs>) {
+    if (vnode.attrs.ajaxOperationMonitor) {
+      this.ajaxOperationMonitor = vnode.attrs.ajaxOperationMonitor;
+    }
+  }
+
   view(vnode: m.Vnode<Attrs>) {
     const isSmall    = vnode.attrs.small;
     const isDropdown = vnode.attrs.dropdown;
     let clickHandler = vnode.attrs.onclick;
+
     if (vnode.attrs.ajaxOperation) {
       clickHandler = (e: MouseEvent) => {
-        if (this.ajaxInProgress()) {
+        if (this.ajaxOperationMonitor!() === OperationState.IN_PROGRESS) {
```

Comment:
as `this.ajaxOperationMonitor` is already initialized I don't think `!` is required here and other places

Reply:
Fixed